library/src/main/java/com/mapzen/valhalla/Router.kt

```diff
@@ -3,14 +3,14 @@ package com.mapzen.valhalla
 interface Router {
 
     enum class Language(private val languageTag: String) {
-        CS_CZ("cs-CZ"),
-        DE_DE("de-DE"),
-        EN_US("en-US"),
-        PIRATE("en-US-x-pirate"),
-        ES_ES("es-ES"),
-        FR_FR("fr-FR"),
-        IT_IT("it-IT"),
-        HI_IN("hi-IN");
+        CS_CZ("cs"),
```

Comment:
Four character country codes have been restored. When using the default Locale, 4-character code is used if it is currently supported. Otherwise 2-character code is used.

Added `ca-ES` and `sl-SI`.